scripts/webpack.config.js

```diff
@@ -3,10 +3,10 @@ const slsw = require('serverless-webpack');
 const nodeExternals = require('webpack-node-externals');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 
-const { NODE_ENV, SERVICE, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE } = process.env;
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_PROVIDER_STAGE } = process.env;
```

Comment:
delete this file?

Reply:
Very likely going to be deleted. There is a ticket next sprint to clean up redundant packages and config.